subsys/bluetooth/shell/flash.c

```diff
@@ -73,17 +79,17 @@ static int cmd_flash(int argc, char *argv[])
 
 	flash_dev = device_get_binding(FLASH_DEV_NAME);
 	if (!flash_dev) {
-		printk("Nordic nRF5 flash driver was not found!\n");
+		error(shell, "Nordic nRF5 flash driver was not found!\n");
```

Comment:
I think `error` macro already adds `\r\n` at the end of a message.

So you will have one extra empty line.